docs/modules/ROOT/pages/manage-api-key.adoc

```diff
@@ -0,0 +1,12 @@
+= Manage Your API Key
+:description: Your API key provides the credentials needed to authenticate clients to your account.
+
+{description}
+
+When you generate your API key, you can copy it directly from the Hazelcast {hazelcast-cloud} console.
```

Comment:
Perhaps it would make sense to add a note with recommendation to save it now, because later on API secret will be hidden and user will see only API Key
<img width="472" alt="image" src="https://github.com/hazelcast/cloud-docs/assets/6086036/593961ef-2def-452b-9e71-707781049400">

Reply:
removed "you can" from this sentence - and added a TIP to the Copy section with the extra information.